genie-common/src/test/java/com/netflix/genie/common/dto/ClusterCriteriaUnitTests.java

```diff
@@ -46,15 +46,6 @@ public void canConstruct() {
         Assert.assertThat(cc.getTags(), Matchers.is(tags));
     }
 
-    /**
```

Comment:
Whatever the expected behavior is (including NPE), isn't worth leaving this here and testing (so that when things change, a flag is raised)

Reply:
You can't pass in null the compiler will fail the build.